Tasks/GradleV2/package.json

```diff
@@ -8,13 +8,12 @@
     "url": "https://github.com/Microsoft/vso-agent-tasks/issues"
   },
   "dependencies": {
+    "@types/node": "^6.0.31",
```

Comment:
6.0.101 is the version that other tasks are using, can you use that?

Reply:
Updated